extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -183,6 +190,10 @@ export class DeployConfigPage extends DacFxConfigPage {
 			return false;
 		}
 		let values = await this.getDatabaseValues();
+		// If there are no databases, call the disableUpgradeRadioButton function.
+		if (values[0] === undefined) {
```

Comment:
does checking if values is undefined not work? Or check if values.length === 0

Reply:
I updated it to values.length === 0 and it still works the same.